js/index.js

```diff
@@ -1 +1,38 @@
-// Your code goes here
+
+
+let bus = document.querySelector('.logo-heading');
+bus.addEventListener('mouseover', function(){ 
+bus.style.color = 'orange'; 
+});
+
+document.addEventListener('keydown', logKey);
+function logKey(e) {
+log.textContent += ` ${e.code}`;
+}
+
+var syntheticEvent = new WheelEvent("syntheticWheel", {"deltaX": 4, "deltaMode": 0});
```

Comment:
Cool use of synthetic events, next step is to apply them to an element! You can use `element.dispatchEvent()` to dispatch a synthetic event